pages/background.md

```diff
@@ -1,25 +1,49 @@
+[Home](https://michaelrappa.github.io/usa-migration/) |
+[Analysis](https://michaelrappa.github.io/usa-migration/pages/analysis.html) |
+[Background](https://michaelrappa.github.io/usa-migration/pages/background.html) |
+[Reflections](https://michaelrappa.github.io/usa-migration/pages/reflections.html) |
+[Workflow](https://michaelrappa.github.io/usa-migration/pages/workflow.html) |
+[Credits](https://michaelrappa.github.io/usa-migration/pages/credits.html) |
+[Bibliography](https://michaelrappa.github.io/usa-migration/pages/bibliography.html) |
+[Sources](https://michaelrappa.github.io/usa-migration/pages/sources.html)
+
+
+
 # Project Background
 
+![Immigration]({{ site.baseurl }}/assets/images/Opendomainimmigrationpic.jpg)
+
+
 ---
+> "The United States is the only major world power
+ today that owes its existence primarily to the phenomenon of
+ immigration" - Ernest Rubin
+ 
+Our project will look at immigration into the United States. We will look at the major countries that have come to the United States and use data visualization to analyze trends in relation to major world events and immigration laws. The project will look at immigration laws and world events to see patterns or trends in the data and hope to come to conclusions with it. We will be using data found from credible sources and plugging them into Tablaeu to create the visualizations. 
```

Comment:
Great, why not use present tense, sense you have (by now) completed the work of the project?